src/screens/SignUp/index.js

```diff
@@ -1,16 +1,25 @@
-import React, { useState } from 'react';
-import './style.scss';
-import plantImg from './assets/growing.jpg';
+import React, { useState } from "react";
+import "./style.scss";
+import plantImg from "./assets/growing.jpg";
 
 const Signup = () => {
   const [user, setUser] = useState({
-    name: '',
-    email: '',
-    password: '',
-    passwordConfirm: '',
+    name: "",
+    email: "",
+    password: "",
+    passwordConfirm: "",
+    isPasswordShown: false,
```

Comment:
A suggestion. You can name variables like `showPassword` and `showConfirmPassword`

Reply:
Updated, thanks!